omod/src/main/webapp/fragments/field/personRelationship.gsp

```diff
@@ -0,0 +1,45 @@
+<%
+    ui.includeJavascript("uicommons", "angular.js")
+    ui.includeJavascript("uicommons", "angular-common.js")
+    ui.includeJavascript("uicommons", "angular-resource.min.js")
+    ui.includeJavascript("uicommons", "angular-ui/ui-bootstrap-tpls-0.6.0.min.js")
+    ui.includeJavascript("uicommons", "services/personService.js")
+    ui.includeJavascript("registrationapp", "field/personRelationship.js")
+%>
+
+<div ng-app="personRelationships" ng-controller="PersonRelationshipController">
+
+    <div ng-repeat="relationship in relationships">
+        <p class="left">
+            <select name="relationship_type" class="rel_type">
+                <option value="">Select Relationship Type</option>
```

Comment:
I mean that instead of hard coding the text "Select Relationship Type", why not use a key in a message properties file?